src/coreclr/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -2284,6 +2288,21 @@ private int GatherClassGCLayout(TypeDesc type, byte* gcPtrs)
                 {
                     result += MarkGcField(fieldGcPtrs, gcType);
                 }
+
+                if (isInlineArray)
+                {
+                    Debug.Assert(field.Offset.AsInt == 0);
+                    int totalLayoutSize = type.GetElementSize().AsInt / PointerSize;
+                    int elementLayoutSize = fieldType.GetElementSize().AsInt / PointerSize;
+                    for (int offset = elementLayoutSize; offset < totalLayoutSize; offset += elementLayoutSize)
+                    {
+                        Buffer.MemoryCopy(gcPtrs, gcPtrs + offset, elementLayoutSize, elementLayoutSize);
+                        result += elementLayoutSize;
+                    }
+
+                    // value array has only one element field
```

Comment:
Need to replace with "inline array".
I started with using "value array" initially in internal references to the feature, but it turned out we have existing things called ValueArray or similar, so I switched to "InlineArray", but missed a few comments.